.travis.yml

```diff
@@ -29,6 +29,8 @@ env:
   - TEST_SUITE=test/unit-test/unit_test.sh
```

Comment:
instead of giving another command to script stage, it is better to put `bash script/call_graph.sh` at the end of `unit_test.sh` itself.